test/geometry_unittest.cpp

```diff
@@ -82,6 +82,14 @@ TEST_CASE("saber::geometry::point", "[saber]")
 		REQUIRE(point2.X() == 1);
 		REQUIRE(point2.Y() == 2);
 	}
+
+	SECTION("Structured Binding")
+	{
+		saber::geometry::Point<int> point{1,2};
+		const auto [x, y] = point; // accessor: structured binding!
+		REQUIRE(x == 1);
+		REQUIRE(y == 2);
+	}
```

Comment:
Add 3 more tests in this section using: `std::tie()` instead of `auto [x, y]`.
Carefully note the difference between `std::tie` and binding notation. See:
https://en.cppreference.com/w/cpp/utility/tuple/tie
You need a test each for:
1. `std::tie(x, y)`
2. `std::tie(x, std::ignore)`
3. `std::tie(std::ignore, y)`